drivers/hmis/app/models/hmis/form/assessment_processor.rb

```diff
@@ -76,11 +81,15 @@ def enrollment_coc_factory(create: true)
   def exit_factory(create: true)
     return self.exit if self.exit.present? || !create
 
-    self.exit = enrollment_factory.build_exit(
-      personal_id: enrollment_factory.client.personal_id,
-      user_id: assessment_detail.assessment.user_id,
-      exit_date: assessment_detail.assessment.assessment_date,
-    )
+    if enrollment_factory.exit.present?
+      # Enrollment already has an Exit that's not tied to this processor (could occur in imported data..)
+      self.exit = enrollment_factory.exit
+    else
+      self.exit = enrollment_factory.build_exit(
+        personal_id: enrollment_factory.client.personal_id,
```

Comment:
unless I'm mistaken, `enrollment_factory.personal_id` should be the same as the client's

Reply:
Yes, I think you're right. Just copying what was there